.github/pull_request_template.md

```diff
@@ -1,50 +1,77 @@
-<!--- Provide a summary of your changes in the title field above. For guidance on formatting, see the comment at the bottom of this template. -->
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
 
-# Description
+Search open/closed issues before submitting. Someone may have pushed the same thing before!
 
-<!--- Describe your changes. -->
+Provide a summary of your changes in the title field above. For guidance on title formatting, see the comment at the bottom of this template.
+-->
 
-## Motivation & context
+# Pull Request
 
-<!--- What problem does this change solve? -->
-<!--- Provide a link if you are addressing an open issue. -->
+## 📖 Description
 
-## Issue type checklist
+<!---
+Provide some background and a description of your work.
+What problem does this change solve?
+Is this a breaking change, chore, fix, feature, etc?
+-->
 
-<!--- What type of change are you submitting? Put an x in the box that applies: -->
+### 🎫 Issues
 
-- [ ] **Chore**: A change that does not impact distributed packages.
-- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
-- [ ] **New feature**: A change that adds functionality.
+<!---
+* List and link relevant issues here.
+-->
 
-**Is this a breaking change?**
-- [ ] This change causes current functionality to break.
+## 👩‍💻 Reviewer Notes
 
-<!--- If yes, describe the impact. -->
+<!---
+Provide some notes for reviewers to help them provide targeted feedback and testing.
 
-**Adding or modifying component(s) in `@microsoft/fast-components` checklist**
+Do you recommend a smoke test for this PR? What steps should be followed?
+Are there particular areas of the code the reviewer should focus on?
+-->
 
-<!-- Do your changes add or modify components in the @microsoft/fast-components package? Put an x in the box that applies: -->
+## 📑 Test Plan
 
-- [ ] I have added a new component
-- [ ] I have modified an existing component
-- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
-- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)
+<!---
+Please provide a summary of the tests affected by this work and any unique strategies employed in testing the features/fixes.
+-->
 
-## Process & policy checklist
+## ✅ Checklist
 
-<!--- Review the list and check the boxes that apply. -->
+### General
 
+<!--- Review the list and put an x in the boxes that apply. -->
+
+- [ ] I have followed the title guidelines for my PR
+- [ ] I have included a change request file using `$ yarn change`
```

Comment:
Assuming we are waiting for the updated publishing work to go in for this? If that is the case, then we don't need the title guidelines. You can title your PR anything once we shift to the `yarn change` process.

Reply:
We can wait until we make the updates. It was an open question in my mind, but let's hold this for the beachball change and bring it all in at the same time. I'll remove the title guideline-related items.